.gitignore

```diff
@@ -263,3 +263,6 @@ dmypy.json
 .history
```

Comment:
Can you add in docs pages information that we support xgb, catboost and lightgbm for shaprfecv?

This would go to both:
 docs/tutorials/nb_shap_feature_elimination.ipynb
README.md

Reply:
This should have been added in previous commits.